src/components/radio/McRadio.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {FormGroup, Input, Label} from 'reactstrap';
+
+const McRadio: React.FC<{radios: Array<{label: string; value: number}>}> = ({radios}) => {
```

Comment:
Podríamos definir los tipos fuera. Quizás se lea mejor. Tb nos permitiría seguir el mismo patrón que en el componente `<Detail>`


```suggestion
type Radio = {
  label: string; 
  value: number;
}

type McRadioProps = {
  radios: radio[]
};
const McRadio = ({radios}: McRadioProps) => {
```

Reply:
refactor done